static/_redirects

```diff
@@ -584,4 +584,4 @@
 /getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/next-steps
 /getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps-* /docs/getting-started/setup-prisma/add-to-existing-project/mongodb/next-steps
 
-
+/getting-started/quickstart /getting-started/quickstart-sqlite
```

Comment:
Super annoying nit I know: could you put this above the `*` redirects? Line 541 would work.

Reply:
We can fix that later, we need this merged in